.github/workflows/check-codestyle.yml

```diff
@@ -47,15 +47,15 @@ jobs:
             -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
             . -B ${{github.workspace}}/build
 
-      - uses: ZedThree/clang-tidy-review@v0.13.4
+      - uses: ZedThree/clang-tidy-review@v0.14.0
         id: review
         with:
-          apt_packages: libboost-dev
+          apt_packages: libboost-dev,python3-dev
```

Comment:
Why do we need `python3-dev`?

Reply:
I'm trying to fix [this](https://github.com/Mstrutov/Desbordante/actions/runs/7253156385/job/19759168907?pr=327#step:6:283) error. `clang-tidy` can't find the `Python.h` header for some reason

Reply:
Ok, it should be added in a separate commit ideally